### PR TITLE
feat: transition to `SvelteKit` and `Vite`

### PR DESCRIPTION
**Pull request by grtlr**
**Still work in progress.**

@fredhohman @domoritz: This changes some of the HTML elements that we use in our Svelte components. Please test the UI shelf and the navigation in the matrix when reviewing this PR. 🙏 

This PR updates to the latest dependencies and thereby fixes the problems of #89.

It consists of two parts:
* [x] Update dependencies and fix accessibility warnings
* [x] Fix Jest test cases which are broken due to `verbatimModuleSyntax`

### CODE REVIEW

**Comment by domoritz** [comment]
@grtlr do you want to upgrade to sveltekit and vite or try to fix this pr as is?

**Comment by grtlr** [comment]
I think we should upgrade to SvelteKit and Vite. I will take a look in the next couple of days!

**Comment by domoritz** [comment]
I'll mark this as a draft for now.

**Comment by grtlr** [comment]
Thanks! Sorry, I have been quite busy the last couple of days, but I plan to spent some more time on it next week.

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "@apple/hierarchical-confusion-matrix",
     "author": "Apple Inc.",
     "version": "1.0.1",
-    "license": "Apple Sample Code",
+    "license": "SEE LICENSE IN `LICENSE`",
```
Comment (domoritz) [inline package.json:5]: What's the reason for this change?

File: public/data/beverages.json
Comment (domoritz) [inline public/data/beverages.json:1]: Why remove this file?

File: src/lib/components/LegendSize.svelte [hunk 2]
```diff
@@ -5,21 +5,38 @@ Copyright (C) 2022 Apple Inc. All Rights Reserved.
 -->
 
 <script>
-    import { scaleLinear } from 'd3-scale';
     import { format } from '../view';
+    import { scaleLinear } from 'd3-scale';
 
     export let scale = d3.scaleLinear();
```
Comment (domoritz) [inline src/lib/components/LegendSize.svelte:11]: is d3 defined here?

**Review by domoritz — APPROVED** [review]
Thanks for the upgrades!

File: package.json (hunk 1 above)
Comment (grtlr) [reply package.json:5]: This gets rid of the warning about incorrect SPDX license during `yarn install`.

File: public/data/beverages.json
Comment (grtlr) [reply public/data/beverages.json:1]: Those files got moved to `static/data`, which I think is the default for SvelteKit.

File: package.json (hunk 1 above)
Comment (fredhohman) [reply package.json:5]: TIL.

**Comment by fredhohman** [comment]
Just took a look - thanks for the updates @grtlr!
